src/platform/utilities/feature-toggles/index.jsx

```diff
@@ -11,29 +14,79 @@ function makeEnvironmentToggleValues(env = environments) {
   };
 }
 
+function getBootstrappedToggleValues() {
+  return {
+    facilityLocatorShowCommunityCares: true,
+  };
+}
+
+const initialToggleValues = {
+  ...makeEnvironmentToggleValues(),
+  ...getBootstrappedToggleValues(),
+};
+
+let currentToggleValues = initialToggleValues;
+const getToggleValues = () => currentToggleValues;
+
+async function connectFeatureToggle(
+  dispatch,
+  toggleValues = currentToggleValues,
+) {
+  dispatch({
+    type: 'FETCH_TOGGLE_VALUES_STARTED',
+    payload: toggleValues,
+  });
+
+  if (!environments.isProduction) {
+    const newToggleValues = await fetchToggleValues();
+
+    dispatch({
+      type: 'FETCH_TOGGLE_VALUES_SUCCEEDED',
+      payload: newToggleValues,
+    });
+  }
+}
+
+addSubscriberCallback(newToggleValues => {
+  currentToggleValues = {
+    ...currentToggleValues,
+    ...newToggleValues,
+  };
+});
+
+const subscribeToToggleUpdates = callback => addSubscriberCallback(callback);
+
 const ToggleContext = React.createContext();
 
 function ToggleProvider(props) {
   const {
     children,
-    initialToggleValues,
+    providerInitialToggleValues,
     subscribeToUpdateToggles = () => {},
     unsubscribeToUpdateToggles = () => {},
   } = props;
 
-  const [toggleValues, setToggleValues] = useState(initialToggleValues);
-
-  function handleToggleValuesUpdate(newToggleValues) {
-    setToggleValues(newToggleValues);
-  }
+  const [toggleValues, setToggleValues] = useState(providerInitialToggleValues);
 
   useEffect(
     () => {
+      function handleToggleValuesUpdate(newToggleValues) {
```

Comment:
the linting complained about this being outside of the effect. It's not used in this demo but worked when moved down here.